community/procedure-compiler/integration-tests/src/test/java/org/neo4j/tooling/procedure/ProcedureTest.java

```diff
@@ -80,6 +80,21 @@ public void calls_procedures_with_simple_input_type_returning_void()
     }
 
     @Test
+    public void calls_procedures_with_different_modes_returning_void()
```

Comment:
Does this test actually test anything? Do session.run() methods throw exceptions on failure?

Reply:
Yes, it will throw exception if you can't execute the query.